where-to-travel/src/main/webapp/style.css

```diff
@@ -1,38 +1,107 @@
+.active {
+  outline: 5px solid #6CC551;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  box-shadow: 6px 12px 9px #888888;
+  z-index: 500;
+}
+
 body {
   height: 100%;
   margin: 0px;
   padding: 0px;
+  transition: background-color 0.5s;
+  color: #1E3C19;
+}
+
+.btn-icon {
+  margin-right: 15px
 }
 
-#container { 
+.card-scroller {
+  background-color: rgba(0, 0, 0, 0);
   height: 100%;
-  width: 100%;
+  left: 0;
+  min-width: 5px;
+  overflow: scroll;
+  position: absolute;
+  top: 0;
+  width: fit-content;
+  z-index: 20;
+}
+
+#colon {
+  font-size: 24px;
+  padding-top: 10%;
+}
+
+#drawer-toggle {
+  color: #049688;
+  font-size: 1.5em;
+}
+
+.fg-mixin {
+  margin: auto;
+  width: 20%
 }
 
-html { 
+html {
   height: 100%;
 }
 
-#input-wrap {
-  display: block;
+
+.icon {
+  background:#fff;
+  border-radius:50%;
+  color:#aaa;
+  cursor:pointer;
+  display:inline-block;
+  margin:0 0 15px;
+  padding:10px 12px 8px;
+  transition:.2s;
+}
+
+.icon:hover {
+  color:#666;
 }
 
-input[type="text"] {
-  width: 30px;
+.icon.press {
+  color:#e23b3b;
+}
+
+.location-card {
+  margin-bottom: 10px;
+  margin-top: 10px;
 }
 
 #map {
   height: 100%;
   width: 100%;
+  z-index: 1;
 }
 
-#nav {
-  display: block;
-  background-color: #fff;
-  border: 1px #000 Solid;
-  height: 100%;
-  padding: 5px;
+#nav-text {
+  margin-right: 10px;
+  margin-left: 10px;
+}
+
+.over-card {
+  bottom: 0;
+  height: fit-content;
+	left: 0;
```

Comment:
Were left and right intentionally indented? What's the reasoning here?

Reply:
Hm, locally they are indented correctly, and git says my branch is up to date. I will try to resolve this before I attribute it to a github display issue

Reply:
fixed with 8e95475